lua/wikis/commons/Widget/Image/Icon/Fontawesome.lua

```diff
@@ -6,20 +6,110 @@
 -- Please see https://github.com/Liquipedia/Lua-Modules to contribute
 --
 
+local Array = require('Module:Array')
 local Class = require('Module:Class')
+local Logic = require('Module:Logic')
 local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
 
+local I = Lua.import('Module:Widget/Html/All').I
```

Comment:
i think for the plugin it would be better to do
```suggestion
local HtmlWidgets = Lua.import('Module:Widget/Html/All')
local I = HtmlWidgets.I
```
or use `HtmlWidgets.I` in the code below and kick the 2nd line here